src/c++/perf_analyzer/client_backend/openai/openai_client.h

```diff
@@ -0,0 +1,180 @@
+// Copyright 2024, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

Comment:
Should these files be called `open_ai_chat_completion_client.h`/`cc`?

Reply:
TMA-1659